Update upgrade notes for Guillotine 7 (#790)
enonic/app-guillotine: docs/release.adoc

WARNING: This release includes <<breaking_changes, breaking changes>>

> IMPORTANT: All `media` and `content` URLs in the `pageUrl`, `imageUrl`, `mediaUrl`, `attachmentUrl` fields and in the `processedHtml` field of <<api#_richtext,RichText>> will be generated as an `absolute` (`server`) URL, relative to the context of your endpoint. If you want to use a different domain or base URL you should implement your own custom processing.

Reviewer: ```suggestion
IMPORTANT: All `media` and `content` URLs in the `pageUrl`, `imageUrl`, `mediaUrl`, `attachmentUrl` fields and in the `processedHtml` field of <<api#_richtext,RichText>> will be `absolute` (`server`) URLs generated in the context of your endpoint. If you want to use a different domain or base URL you should implement your own custom processing.
```

> IMPORTANT: All links/urls to `media` or `content` within the `pageUrl`, `imageUrl`, `mediaUrl`, `attachmentUrl` fields and inside the `processedHtml` field of <<api#_richtext,RichText>> will be generated as an `absolute` (`server`) link relative to the context of your endpoint. For instance, if you want to use a different domain or base URL you should make custom processing by your client.

Reviewer: ```suggestion
IMPORTANT: All `media` and `content` URLs in the `pageUrl`, `imageUrl`, `mediaUrl`, `attachmentUrl` fields and in the `processedHtml` field of <<api#_richtext,RichText>> will be generated as an `absolute` (`server`) URL, relative to the context of your endpoint. If you want to use a different domain or base URL you should implement your own custom processing.
```
